src/ol/geom/linestring.js

```diff
@@ -203,6 +204,20 @@ ol.geom.LineString.prototype.intersectsExtent = function(extent) {
 
 
 /**
+ * @param {number} offset Offset.
+ * @param {number} count Count.
+ * @param {Array.<ol.Coordinate>=} opt_coordinates Coordinates.
+ * @api
+ */
+ol.geom.LineString.prototype.replaceCoordinates =
```

Comment:
Do I put the examples in the comment or create a new file under the `/examples/` directory?